CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 
 ### Internal
 
+- [In Progress: 22 Sep 2021] Custom Drug Entry Sheet UI Improvements
+- [In Progress: 31 Aug 2021] Refactor logic around providing drug frequencies label depending on the country
```

Comment:
> Refactor logic around providing drug frequencies label depending on the country

Same with this one. This is already completed and part of the changelog.

Reply:
Removed this in favour of the changelog entries that mention what the refactoring is (providing as a map or injecting into specific screens/view etc.,)